.buildkite/commands/checkout-release-branch.sh

```diff
@@ -1,11 +1,6 @@
 #!/bin/bash -eu
 
-# RELEASE_VERSION is passed as an environment variable from fastlane to Buildkite
-#
-if [[ -z "${RELEASE_VERSION}" ]]; then
-    echo "RELEASE_VERSION is not set."
-    exit 1
-fi
+RELEASE_VERSION="${1?Please provide a release version as an argument.}"
```

Comment:
TIL about the `${parameter?word}` expansion with `set -u` combo.

From `bash` manpage

> When not performing substring expansion, using the forms documented below (e.g., :-), bash tests for a parameter that is unset or null.  Omitting the colon results in a test only for a parameter that is unset.
>
> [...]
>
> `${parameter:?word}` Display Error if Null or Unset.  If parameter is null or unset, the expansion of word (or a message to that effect if word is not present) is written to the standard error and the shell, if it is not interactive, exits.  Otherwise, the value of parameter is substituted.

Reply:
Nitpick: I find it DRY to move the Buildkite header in the script:
```suggestion
RELEASE_VERSION="${1?Please provide a release version as an argument.}"

echo '--- :git: Checkout Release Branch'

```

Reply:
> TIL about the `${parameter?word}` expansion with `set -u` combo.
> 
> From `bash` manpage

My favorite cheat sheet I use as reference for those kind of things: https://devhints.io/bash